### PR TITLE
Removing ab high relevance switch

### PR DESCRIPTION
**Pull request by uplne**
This is not being used anymore

### CODE REVIEW

File: data/database/dec28e54a17cb1fadaaac2ecb387469d34e093b1698b6e7df5811e840b296b18 [hunk 1]
```diff
@@ -0,0 +1 @@
+{"response":{"status":"ok","userTier":"internal","total":0,"startIndex":0,"pageSize":1,"currentPage":1,"pages":0,"orderBy":"newest","tag":{"webTitle":"Crossword","id":"type/crossword","type":"type","webUrl":"http://www.theguardian.com/crossword","apiUrl":"http://content.guardianapis.com/type/crossword","references":[]},"results":[],"leadContent":[]}}
```
Comment (johnduffell) [inline data/database/dec28e54a17cb1fadaaac2ecb387469d34e093b1698b6e7df5811e840b296b18]: is this related?

**Comment by johnduffell** [comment]
:+1: to the switch but not sure about the random file?

File: data/database/dec28e54a17cb1fadaaac2ecb387469d34e093b1698b6e7df5811e840b296b18 (hunk 1 above)
Comment (NataliaLKB) [inline data/database/dec28e54a17cb1fadaaac2ecb387469d34e093b1698b6e7df5811e840b296b18]: @johnduffell As far as I know this is generated when you run the Scala Tests.

Comment (johnduffell) [inline data/database/dec28e54a17cb1fadaaac2ecb387469d34e093b1698b6e7df5811e840b296b18]: the build (should) fail if it's not present and it's needed - I'm just wondering if this was deleted by someone and now it's being accidentally added back in.

Comment (uplne) [inline data/database/dec28e54a17cb1fadaaac2ecb387469d34e093b1698b6e7df5811e840b296b18]: I will check if it is still needed
